README.md

```diff
@@ -6,7 +6,7 @@
 [![crates.io](https://img.shields.io/crates/v/der-parser.svg)](https://crates.io/crates/der-parser)
 [![Download numbers](https://img.shields.io/crates/d/der-parser.svg)](https://crates.io/crates/der-parser)
 [![dependency status](https://deps.rs/crate/der-parser/9.0.0/status.svg)](https://deps.rs/crate/der-parser/9.0.0)
-[![Github CI](https://github.com/rusticata/der-parser/workflows/Continuous%20integration/badge.svg)](https://github.com/rusticata/der-parser/actions)
+[![Github CI](https://github.com/rusticata/der-parser/actions/workflows/rust.yml/badge.svg)](https://github.com/rusticata/der-parser/actions/workflows/rust.yml)
```

Comment:
This change must be reflected in `src/lib.rs` as well.
In fact, the process is that I update `src/lib.rs` and run `cargo sync-readme` to update the part of `README.md` inside the tags
```
<!-- cargo-sync-readme start -->
...
<!-- cargo-sync-readme end -->
```

My bad, it's not documented anywhere :/

Reply:
Note that you do not need to install the tool, just keeping the files in sync should be enough.
The goal is to avoid overwriting changes next time I will run `cargo sync-readme` :)

Reply:
np, I will fix :-)

Reply:
Fixed both the badge update & the MSRV update.